tests/deploy_test.go

```diff
@@ -13,6 +14,88 @@ import (
 
 var emptyQueryString = ""
 
+func Test_Scaling_Up(t *testing.T) {
+
+	functionRequest := requests.CreateFunctionRequest{
+		Image:      "functions/alpine:latest",
+		Service:    "test-scaling",
+		Network:    "func_functions",
+		EnvProcess: "env",
+	}
+
+	deployStatus, deployErr := deploy(t, functionRequest)
+	if deployErr != nil {
+		t.Errorf(deployErr.Error())
+		t.Fail()
+		return
+	}
+
+	if deployStatus != http.StatusOK && deployStatus != http.StatusAccepted {
+		t.Errorf("got %d, wanted %d or %d", deployStatus, http.StatusOK, http.StatusAccepted)
+	}
+
+	invoke(t, functionRequest.Service, emptyQueryString, http.StatusOK)
+
+	err := scale(t, functionRequest.Service, "10")
+	if err != nil {
+		t.Errorf(err.Error())
+		t.Fail()
+		return
+	}
+
+	replicas, err := getReplicas(t, functionRequest.Service)
+	if err != nil {
+		t.Errorf(err.Error())
+		t.Fail()
+		return
+	}
+
+	want := "10"
+	if want != fmt.Sprintf("%d", replicas) {
+		t.Errorf("Want %s replicas, Got %d", want, replicas)
+	}
+
+}
+
+func scale(t *testing.T, fnName string, replicas string) error {
+
+	request := `{"service": "` + fnName + `", "replicas": ` + replicas + `}`
+	reqBody := bytes.NewBufferString(request)
+
+	endpoint := os.Getenv("gateway_url") + "system/scale-function/" + fnName
+	res, err := http.Post(endpoint, "application/json", reqBody)
+	if err != nil {
+		return fmt.Errorf("Failed to POST request %s to endpoint %s. Error: %t", request, endpoint, err)
+	}
+
+	if res.StatusCode != 200 && res.StatusCode != 202 {
```

Comment:
Rather than integers use `http.StatusAccepted` etc

Reply:
Fixed in PR#24.